src/vs/workbench/contrib/tasks/common/taskConfiguration.ts

```diff
@@ -1795,7 +1795,7 @@ namespace TaskParser {
 	}
 }
 
-interface Globals {
+export interface Globals {
```

Comment:
Removing all these exports that we can would be good, otherwise they pollute the global namespace/auto import results

Reply:
is there a way around this?

Reply:
they're all used

Reply:
Guess not if they're all used.

Can you think of a better name for `Globals`? It's not very clear from looking at the code what it's meant to be exactly.